docs/source/modules/simcloud.rst

```diff
@@ -0,0 +1,65 @@
++--------------------------+-----------------------------------------------------+--------------------+-------------------------------+
+|``reff_rad``              | Effective cloud particle radius                     | :math:`\mu` m      | (pfull, lat, lon)             |
++--------------------------+-----------------------------------------------------+--------------------+-------------------------------+
+|``qcl_rad``               | Specific humidity of cloud liquid                   | kg kg :math:`^{-1}`| (phalf, lat, lon)             |
++--------------------------+-----------------------------------------------------+--------------------+-------------------------------+
+|``rh_in_cf``              | RH as a percent                                     | :math:`\%`         | (phalf, lat, lon)             |
++--------------------------+-----------------------------------------------------+--------------------+-------------------------------+
+
+Cloud Coverage
+^^^^^^^^^^^^^^
+The module ``cloud_cover`` provides additional diagnostics to show spatial coverage of clouds in the lat-lon direction; additionally split into different levels of clouds.
+
++------------------+-----------------------------+------------+-------------------------------+
+| Name             | Description                 | Units      | Dimension (not including time)|
++==================+=============================+============+===============================+
+|``tot_cld_amt``   | Total cloud amount (%)      | :math:`\%` | (lat, lon)                    |
++------------------+-----------------------------+------------+-------------------------------+
+|``high_cld_amt``  | High level cloud amount (%) | :math:`\%` | (lat, lon)                    |
++------------------+-----------------------------+------------+-------------------------------+
+|``mid_cld_amt``   | Hid-level cloud amount (%)  | :math:`\%` | (lat, lon)                    |
++------------------+-----------------------------+------------+-------------------------------+
+|``low_cld_amt``   | Low level cloud amount (%)  | :math:`\%` | (lat, lon)                    |
++------------------+-----------------------------+------------+-------------------------------+
+
+Marine Stratospheric Clouds
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+There are additional diagnostics specifically relating to marine stratospheric clouds available in the module ``strat_cloud``. These are primarily available for test purposes so are not detailed here, however they may be accessed in the routine ``marine_strat_cloud.F90`` within the cloud scheme directory.
+
+
+Relevant modules and subroutines
+--------------------------------
+The SOCRATES radiation scheme is setup in ``src/atmos_param/socrates/interface/``.
+
+References
+----------
+[Liu2021]_
+
+Authors
+-------
+This documentation was written by Daniel Williams (with thanks to Qun Liu for implementing the model).
```

Comment:
update to:
"This documentation was written by Daniel Williams and reviewed by Penelope Maher and X." Is Neil listed to review too?
I then suggest you add:

"The SimCloud scheme was implemented by Qun Liu who greatly improved the cloud scheme implemented by Penelope Maher and Stephen Thomson."